File: projekat/src/app/entities/korisnik/korisnik.ts
export class Korisnik {
    email: string;
    lozinka: string;
    ime: string;
    prezime: string;
    grad: string;
    telefon: string;
    prijatelji: string;

    constructor(email: string, lozinka: string, ime: string, prezime: string,grad: string, telefon: string ,prijatelji: string) {
        this.email = email;
        this.lozinka = lozinka;
        this.ime = ime;
        this.prezime = prezime;
        this.grad = grad;
        this.telefon = telefon;
        this.prijatelji = prijatelji;
    }

}

File: WebApplication1/WebApplication1/Models/Korisnik.cs
using Microsoft.AspNetCore.Identity;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication1.Models
{
    public enum TipKorisnika
    {
        Neregistrovani,
        Registrovani,
        AdminAvioKompanije,
        AdminRentACarServisa,
        Administrator
    }

    public class Korisnik : IdentityUser
    {
        //[Key]
        //public int Id { get; set; }

        //public string Email { get; set; }

        //public string Lozinka { get; set; }

        public string Ime { get; set; }

        public string Prezime { get; set; }

        public string Grad { get; set; }

        public string Telefon { get; set; }

        public TipKorisnika Uloga { get; set; }

        public bool IzmenjenaLozinka { get; set; }
    }
}
